docs/content/documentation/getting-started/configuration.md

```diff
@@ -41,6 +41,12 @@ generate_rss = false
 # not set (the default).
 # rss_limit = 20
 
+# Whether to copy or hardlink files in static/ directory. Useful for sites
+# whose static files are large. Note that for this to work, both static/ and
+# output directory need to be on the same filesystem. Also, theme's static/
+# files are always copies, regardles of this setting. False by default.
+# hard_link_static = true
```

Comment:
this doc shows the default so it should be false here

Reply:
Changed.